### ISSUE TITLE
Questions on backward.cu

### ISSUE THREAD

**Issue by AmeYatoro** [graphdeco-inria/gaussian-splatting#797]
I've noticed that backward.cu implemented the computation of gradients, while it is not used in class GaussianRasterizer, does this mean it is actually useless and the backprop of 3DGS depends completely on pytorch autograd system as loss.backward()?

**Comment by lingxin12** [comment]
Please google search: torch.autograd.Function class, and learn it.
